patroni/postgresql/__init__.py

```diff
@@ -135,17 +135,25 @@ def lsn_name(self):
 
     @property
     def cluster_info_query(self):
-        pg_control_timeline = 'timeline_id FROM pg_catalog.pg_control_checkpoint()' \
-                if self._major_version >= 90600 and self.role == 'standby_leader' else '0'
+        if self._major_version >= 90600:
+            extra = (", CASE WHEN latest_end_lsn IS NULL THEN NULL ELSE received_tli END,"
```

Comment:
did you renamed it because `cluster_info_state` has a field of the same name? Maybe there's a better name than `extra`?

Reply:
Previously it was just a single function call, now we have one or two functions and constructing part of the query which are appended to the original query, that's why I chose a name `extra`.